examples/config-boltdb.json

```diff
@@ -0,0 +1,19 @@
+{
+    "distSpecVersion": "1.0.1-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot",
+        "cacheDatabaseDriver": {
+            "name": "boltdb",
+            "cacheName": "cache1",
+            "rootDirectory": "/tmp/zot/cachedb",
+            "useRelPaths": "False"
```

Comment:
shouldn't this be bool?

Reply:
Rewrote config and CreateCacheDatabaseDriver to use interface instead, and used type assertion. Also updated the config